woocommerce/woocommerce-admin
# Render optional children in summary number component

As mentioned in https://github.com/Automattic/woocommerce-payments/issues/962 we decide to add a new optional prop to the `SummaryNumber` component, instead of creating a custom one, to support additional details like dates.

I named the prop `details` because we use it not only for dates but also for additional details like `N deposits`. Let me know if you prefer to use another name.

For the placeholder, I choose to not include the markup, because it is optional and not used for now in WC Admin, but I can add it if you prefer.

### Screenshots
*Storybook*
<img src="https://user-images.githubusercontent.com/7670276/113714242-7c7bcf80-96e8-11eb-9fa2-e6722880bde6.png" width ="931" />

*Storybook mobile*
<img src="https://user-images.githubusercontent.com/7670276/113714371-99b09e00-96e8-11eb-8398-85d833b1e82e.png" width="310" />

*WC Pay use case*
<img src="https://user-images.githubusercontent.com/7670276/113709866-4f78ee00-96e3-11eb-8946-d488ac8ca819.png" width="746" />


### Detailed test instructions:

- Run `npm run storybook`
- Check http://localhost:6007/?path=/story/woocommerce-admin-components-summarylist--basic
